index.html

```diff
@@ -1165,25 +1209,58 @@ <h3>Issuer</h3>
 machine-readable information about the <a>issuer</a> that can be used to
 <a>verify</a> the information expressed in the <a>credential</a>.
           </dd>
+        </dl>
+
+        <pre class="example nohighlight" title="Usage of issuer property">
+{
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1",
+    "https://example.com/examples/v1"
+  ],
+  "id": "http://example.edu/credentials/3732",
+  "type": ["VerifiableCredential", "UniversityDegreeCredential"],
+  <span class="highlight">"issuer": "https://example.edu/issuers/14"</span>,
+  "issuanceDate": "2010-01-01T19:23:24Z",
+  "credentialSubject": {
+    "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+    "degree": {
+      "type": "BachelorDegree",
+      "name": "Bachelor of Science in Mechanical Engineering"
+    }
+  },
+  "proof": { ... }
+}
+        </pre>
+      </section>
+
+      <section>
+        <h3>Issuance Date</h3>
+
+        <p>
+The date and time when the <a>credential</a> was issued MUST be
```

Comment:
The date and time when the credential was issued are expressed using the following property:
 
 issuanceDate

A verifiable credential MUST contain an issuanceDate property. The value of the issuanceDate property MUST be a string value . . .